jsapp/js/components/common/checkbox.stories.tsx

```diff
@@ -1,17 +1,23 @@
-import React from 'react'
-
-import type { ComponentMeta, ComponentStory } from '@storybook/react'
+import type { Meta, StoryObj } from '@storybook/react'
 import Checkbox from './checkbox'
 
-export default {
+const meta: Meta<typeof Checkbox> = {
   title: 'common/Checkbox',
   component: Checkbox,
-  argTypes: {},
-} as ComponentMeta<typeof Checkbox>
+  argTypes: {
+    label: { type: 'string' },
+    checked: { type: 'boolean' },
+    onChange: { action: 'changed' },
+  },
+}
+
+export default meta
```

Comment:
how about `export default {} satisfies Meta<typeof Checkbox>`? Would simplify away a variable.

Reply:
nevermind, storybook crashes that way.